book/src/developers.md

```diff
@@ -0,0 +1,3 @@
+# Developers
+
+[Developer documentation (external link)](https://docs.rs/goose/)
```

Comment:
Remove, appropriate links will be added in followup when we focus on properly structuring this book.